ios/Video/RCTVideo.m

```diff
@@ -530,7 +531,6 @@ - (void)playerItemForSource:(NSDictionary *)source withCallback:(void(^)(AVPlaye
 - (void)playerItemForSourceUsingCache:(NSString *)uri assetOptions:(NSDictionary *)options withCallback:(void(^)(AVPlayerItem *))handler {
     NSURL *url = [NSURL URLWithString:uri];
     [_videoCache getItemForUri:uri withCallback:^(RCTVideoCacheStatus videoCacheStatus, AVAsset * _Nullable cachedAsset) {
-        switch (videoCacheStatus) {
```

Comment:
Oh I missed this, this must be a typo no? You can't remove the switch case like that? @samermurad

Reply:
@jenshandersson oh wow yeah this is definitely a typo, I'll Fix it in a sec

Reply:
@jenshandersson Fixed